src/components/Header/Header.js

```diff
@@ -1,35 +1,39 @@
 import React from 'react';
 import HeaderNav from './HeaderNav/HeaderNav';
+import HeaderMessages from './HeaderMessages/HeaderMessages';
 import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
 import headerData from '../../headerData.json';
 
 const Header = (props) => {
   return (
-    <header className="header-main">
-      <div className="header-wrapper">
-        <div className="logo">
-          <span className="ms-icon icon-doppler-logo" />
+    <div>
+      {headerData.alert ? <HeaderMessages alert={headerData.alert} /> : null}
+      <header className="header-main">
+        <div className="header-wrapper">
+          <div className="logo">
+            <span className="ms-icon icon-doppler-logo" />
+          </div>
+          <HeaderNav nav={headerData.nav} />
+          <nav className="nav-right-main">
+            <ul>
+              <li>
+                <a href="https://www.fromdoppler.com/" className="active" data-count="1">
```

Comment:
@fcoronelmakingsense please note you have links that have english and spanish resources: 
english: `https://www.fromdoppler.com/en/`
spanish: `https://www.fromdoppler.com/` 
english: `https://help.fromdoppler.com/en/` 
spanish: `https://help.fromdoppler.com/es/`

Reply:
Fixed